docs/developer/guidelines/dashboards.md

```diff
@@ -42,63 +42,83 @@ Ensure logos render correctly on the Dashboard List page and within the preset d
-- [x] Well formatted, concise notes
-- [x] Color coordination between related groups, notes within groups, and graphs within groups
-
-Follow the guidelines below as you build your dashboard to achieve something similar.
+- [x] Well formatted, concise notes explaining the value or purpose of data in each group. Try the presets "caption", "annotation", or "header", or pick your own combination of styles. Avoid using the smallest font size for notes that are long or include complex formatting, like bulleted lists or code blocks.
+- [x] All widgets are placed within a group based on thematic organization, rather than directly on the background of the dashboard
+   ![Group titles](../assets/images/group_titles.png)
+- [x] *Query value* widgets have a Timeseries background (e.g. "Bars") instead of being blank
+- [x] Visualizations with obvious thresholds or zones use semantic formatting for graphs or custom red/green/yellow text formatting for query values. 
+- [x] Color coordination between group headers, notes within groups, and graphs within groups (e.g. all group headers or note widgets the same colour). If you've applied a vivid green to all group headers, try making its notes light green as well.
+       ![Header color usage](../assets/images/header_color_usage.png)
+- [x] Legends for each graph. Legends make it easy to read a graph without having to hover over each series or maximize the widget. Make sure you use aliases so the legend is easy to read. Automatic mode for legends is a great option that hides legends when space is tight and shows them when there's room.
+   ![Use legends](../assets/images/use_legends.png)
+- [x] Adjascent graphs have aligned x-axes. If one graph is showing a legend and the other isn't, the x-axes won't align - make sure they either both show a legend or both do not.
+   ![X axes alignment](../assets/images/x_axes_alignment.png)
+- [x] For timeseries, base the display type on the type of metric
+    | Types of metric | Display type |
+    | - | - |
+    | Volume (e.g. Number of connections) | `area` |
+    | Counts (e.g. Number of errors) | `bars` |
+    | Multiple groups or default | `lines` |
 
-### General
+### Creating a New Dashboard
 
-1. When creating a new dashboard, select the default dashboard type (internally called multisize layout).
+1. After selecting *New Dashboard*, you will have the option to choose between: *Dashboard, Screenboard*, and *Timeboard*. *Dashboard* is recommended.
 
-1. Dashboard titles should contain the integration name. Some examples of a good dashboard title are `Syclla` and `Cilium Overview`.
+1. Add a logo to the dashboard header. The integration logo will automatically appear in the header if the icon exists [here](http://app.datadoghq.com/internal/styleguide/logos/logo-gallery?variant=avatar) and the `integration_id` matches the icon name. That means it will only appear when the dashboard you're working on is made into the official integration board.
```

Comment:
is this link still accurate? should it be `https://druids.datadoghq.com/logos/logo-gallery`?

Reply:
Oop yes thanks for the catch. Should be updated with druids link 👍